### PR TITLE
Make locales configurable

### PR DESCRIPTION
**Pull request by shinji-yoshida**
This pull request has 2 changes:

1. Add `locales` argument to `Configure()`
1. Change `logMissing` type from `bool` to `bool?` in order to skip `_isLoggingMissing` modification when `null` is given

### CODE REVIEW

**Review by GabLeRoux — APPROVED** [review]
Sweet, thanks for this change 👍 
It looks good to me. Note that I did not take the time to test it yet, but I see you added a unit test that demonstrates how to use it and everything looks fine. I'll let @camsjams give a look too, but this is definitely something we need.

Seems to relate to #2

File: src/I18nUnity/Mgl/I18n.cs [hunk 1]
```diff
@@ -58,23 +58,28 @@ public static string GetLocale()
...
         public static void SetLocale(string newLocale = null)
         {
-            Configure (_localePath, newLocale, _isLoggingMissing);
+            Configure (newLocale: newLocale);
         }
 
         public static void SetPath(string localePath = null)
         {
-            Configure (localePath, _currentLocale, _isLoggingMissing);
+            Configure (localePath: localePath);
```
Comment (camsjams) [inline src/I18nUnity/Mgl/I18n.cs:66]: :+1:

File: src/I18nUnity/Mgl/I18n.cs [hunk 2]
```diff
@@ -58,23 +58,28 @@ public static string GetLocale()
...
             if (newLocale != null) {
                 _currentLocale = newLocale;
             }
-            _isLoggingMissing = logMissing;
+            if (logMissing.HasValue) {
+                _isLoggingMissing = logMissing.Value;
+            }
+            if (locales != null) {
+                I18n.locales = (string[])locales.Clone ();
+            }
```
Comment (camsjams) [inline src/I18nUnity/Mgl/I18n.cs:82]: nice!

**Review by camsjams — APPROVED** [review]
Looks good! :shipit:

**Pull request merged**